modules/oe_content_featured_media_field/tests/src/FunctionalJavascript/FeaturedMediaFieldWidgetTest.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_featured_media_field\FunctionalJavascript;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\file\Entity\File;
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\media\Entity\Media;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Drupal\Tests\node\Traits\NodeCreationTrait;
+
+/**
+ * Tests the output of "oe_featured_media_widget" widget.
+ *
+ * @group oe_content_featured_media_field
+ */
+class FeaturedMediaFieldWidgetTest extends WebDriverTestBase {
+
+  use ContentTypeCreationTrait;
+  use NodeCreationTrait;
+  use MediaTypeCreationTrait;
+
+  /**
+   * A field storage to use in this test class.
+   *
+   * @var \Drupal\field\Entity\FieldStorageConfig
+   */
+  protected $fieldStorage;
+
+  /**
+   * The field used in this test class.
+   *
+   * @var \Drupal\field\Entity\FieldConfig
+   */
+  protected $field;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'node',
+    'field',
+    'field_ui',
+    'media',
+    'media_test_source',
+    'oe_media',
+    'oe_content_featured_media_field',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->createContentType(['type' => 'page']);
+
+    $user = $this->drupalCreateUser([
```

Comment:
Let's move this login part to the other method, so we can put a comment on which user is logging like we did in line 174.